shared/app/app-state.desktop.tsx

```diff
@@ -64,11 +74,11 @@ export default class AppState {
     }
 
     // Listen to the main window asking for this value
-    SafeElectron.getIpcMain().on('getAppState', event => {
+    getIpcMain().on('getAppState', event => {
```

Comment:
turns out this is super dangerous. the node thread and main thread load this file so they both register for these events